genie-agent/src/main/resources/logback-spring.xml

```diff
@@ -11,7 +11,7 @@
 
     <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
 
-    <property name="LOG_FILE" value="/tmp/genie-agent-${PID}.log"/>
+    <property name="LOG_FILE" value="${GENIE_AGENT_TEMPORARY_LOG_FILE:-/tmp/genie-agent-${PID}.log}"/>
```

Comment:
Not sure the convention here but there is a `:-`. Not sure if it should just be `:`

Reply:
This is `default value` notation like bash or other: https://www.tldp.org/LDP/abs/html/parameter-substitution.html
i.e. use `GENIE_AGENT_TEMPORARY_LOG_FILE` value if set, if not default to `/tmp/genie-agent-${PID}.log`

Reply:
Sounds good. Got it mixed up with Spring parameter substitution